.env.example

```diff
@@ -3,7 +3,9 @@ API_URL=https://agid-apim-prod.azure-api.net/api/v1
 CLIENT_REDIRECTION_URL=/profile.html?token={token}
 CLIENT_ERROR_REDIRECTION_URL=/error.html
 PORT=80
-REDIS_URL="redis://redis"
+REDIS_URL=put_the_azure_redis_url_here
+REDIS_PORT=put_the_azure_redis_port_here
+REDIS_PASSWORD=put_the_azure_redis_password_here
```

Comment:
this should probably be passed by env var

Reply:
yes, the `dotenv` package reads from this file or from the environment variables (on k8s we set those values as environment variables).
In this way a user can run the proxy locally both on the host or in a Docker container